envoy/tracing/trace_context.h

```diff
@@ -66,41 +70,33 @@ class TraceContext {
    * @param key The context key of string view type.
    * @return The optional context value of string_view type.
    */
-  virtual absl::optional<absl::string_view> getByKey(absl::string_view key) const PURE;
+  virtual absl::optional<absl::string_view> get(absl::string_view key) const PURE;
```

Comment:
Removed the unnecessary suffix `ByKey`. We use this suffix to avoid confiliction with methods of `Http::HeaderMap`. But now, the Http::RequestHeaderMap will no longer inherit from `TraceContext`, so we removed it.